### PR TITLE
Advanced search

### PR DESCRIPTION
**Pull request by maelle**
cf #23

### CODE REVIEW

File: R/advanced_search.R [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#' Advanced search of CRAN packages
+#'
```
Comment (maelle) [inline R/advanced_search.R]: this is an empty shell for now, but was this the interface you had in mind @gaborcsardi?

**Comment by gaborcsardi** [comment]
I refactored this. It is currently quite easy to construct a query and is not syntactically valid for Elastic, so maybe we'll want to change the API before the release.

**Comment by maelle** [comment]
Cool, thanks!

How could one validate a query?

**Comment by gaborcsardi** [comment]
I don't think we need to. We just print the error message that we get from the server.

**Pull request merged**